.circleci/config.yml

```diff
@@ -623,7 +623,7 @@ workflows:
   cut-release:
     triggers:
       - schedule:
-          cron: '0 4 * * 3'
+          cron: '0 13 * * 1'
```

Comment:
Changed cron to cut release from Tuesday evening to Monday morning.

Reply:
reaching waaay back for my cron time knowledge..so from Wed 5am to monday 2pm.  What time zone do these run in ?

Reply:
They run in GMT. I simply use https://crontab.guru/